lib/transport_web/router.ex

```diff
@@ -37,6 +37,8 @@ defmodule TransportWeb.Router do
       get "/organizations/form", UserController, :organization_form
       post "/organizations/_create", UserController, :organization_create
       get "/organizations/:slug/datasets/", UserController, :organization_datasets
+      get "/organizations/:slug/datasets/_new", DatasetController, :new
```

Comment:
If we follow the convention of appending a ```_``` only to resources, then ```_new``` should be ```new```.